connexion/apps/flask_app.py

```diff
@@ -21,8 +21,9 @@
 
 
 class FlaskApp(AbstractApp):
-    def __init__(self, import_name, server='flask', **kwargs):
+    def __init__(self, import_name, server='flask', extra_files=None, **kwargs):
```

Comment:
Nit, but all other parameters are documented in the super function. Would be good to document the `extra_files` parameter here? We currently also don't document its type, but assume it's a `list` by calling `.extend()`.

Reply:
True, wasn't sure what happens to the auto-generated documentation but seems that it already doesn't include the documentation of the parent class.
I'll add the `extra_files` documentation and refer to the parent class for the other parameters.